### DISCUSSION TITLE
UI not loading in DAW?

### DISCUSSION THREAD

**Discussion by bbbbbbben** [pierreguillot/Camomile#304] [Q&A]
Is it a common problem for UI not to load in DAW for Camomile plugins? When adding one of these plugins to a channel, a plugin window appears with a loading symbol. I'm using Logic Pro X v10.7.4 on an M1 Mac, and I've had this problem with the example plugins and Mike Moreno's _Lira 8_. 

If anyone has a solution that'd be amazing, I'd love to start making plugins with this amazing tool!

<img width="239" alt="Screenshot 2022-07-20 at 00 23 25" src="https://user-images.githubusercontent.com/68873247/179864504-41c9761e-9b4b-48a5-837f-7143509518fa.png">

**Comment by Josef-N** [comment]
a bit late, but:
Recently I upgraded from Mac Monteray/Intel to Ventura/M2, now I see this issue – no UI in Logic 10.7.9 (also other DAWs). What I found out:

When starting the DAW in Rosetta, then Comomile 1.0.8 b10 plugins load as expected.

I decided to go back to Comomile 1.0.7, as it always worked perfectly for my purposes, and it still does. Even in Ventura/M2, natively (without Rosetta)
